projects/packages/search/src/instant-search/components/search-result-expanded.scss

```diff
@@ -56,10 +60,8 @@ $image-square-size: 128px;
 		display: none;
 	}
 
-	// Aspect ratio box trick
+	// Aspect ratio box trick, pt 2
```

Comment:
That's annoying.

Reply:
Yeah. I tried doing a `& { ... }` block to work around it (which is also the way to signal opt-in to the future behavior), but that triggered stylelint and I decided to just split it up.